lib/__tests__/KTable.spec.js

```diff
@@ -1,5 +1,35 @@
 import { mount } from '@vue/test-utils';
 import KTable from '../KTable';
+import { renderComponentForVisualTest, takeSnapshot } from '../../jest.conf/visual.testUtils';
+
+describe.visual('KTable Border Visual Test', () => {
```

Comment:
Thanks for adding this, much appreciated. I think it'd be fine to simplify a bit and just frame it as 'sortable table' visual test? Because it actually visually tests for many more things than just the border - it shows the table when it's sortable.

Visual tests are a bit different from unit tests in this regard.

Reply:
Makes sense, still new to understanding visual tests here. Made the required change :)